08-graphs/02-graphs-algorithms.md

```diff
@@ -65,9 +172,6 @@ What is the time complexity of Breadth-First-Search with N nodes and E edges.
 
 ##### !end-answer
 
-<!-- other optional sections -->
-<!-- !hint - !end-hint (markdown, users can see after a failed attempt) -->
-<!-- !rubric - !end-rubric (markdown, instructors can see while scoring a checkpoint) -->
 ##### !explanation
 
 Since you will visit each node once, and loop through each of the edges in each node the Big-O of this algorithm is O(N + E) where `N` is the number of nodes in the graph and `E` is the number of edges since each node and each edge will be explored.
```

Comment:
It might be nice to include a more in depth explanation about why O(N+E) is different than O(N*E) or O(N^2) which I've seen as other common responses.

Similarly, we might consider adding O(N^2) as a possible answer